index.js

```diff
@@ -69,6 +68,13 @@ async function openPopup() {
 		notificationsList = select.all('.boxed-group', notificationsPage);
 		if (notificationsList.length === 0) {
 			return;
+		} else if (!select('.notification-indicator .mail-status').classList.contains('unread')) {
+			const newDom = await fetch(document.URL, {
+				credentials: 'include'
+			}).then(r => r.text()).then(domify);
+
+			indicator.parentNode.replaceChild(select('.notification-indicator', newDom), indicator);
+			init();
 		}
```

Comment:
for #18, I tried bringing back the unread indicator this way. 

@bfred-it I'm not sure this is a good way.. any better way to do this?

Reply:
I'll push a commit to this branch in a few minutes